src/actions.js

```diff
@@ -42,7 +42,7 @@ export function addServiceRoute(serviceId, { name, attributes }) {
         type: 'add-service-route',
         endpoint: {name: 'routes', params: {serviceId, routeName: name}},
         method: 'POST',
-        body: assign({}, attributes, { service: { id: serviceId } })
+        body: { name, ...attributes, service: { id: serviceId } }
```

Comment:
Do we need to explicitly add name here since name is in `attributes`? Is this to keep backwards compatibility with Kong < 1.0?

Reply:
We might not, since we're ensuring it's in attributes in `parseRoute`. Unfortunately, I'm not 100% confident that the route argument to these action functions always comes from the output of `parseRoute`. So this might be unnecessary, but I don't think it does any harm to use the top level name in case it's not in attributes.

Reply:
Turns out this was not a wise choice 😅. Adding name to attributes here prevented kongfig from detecting when the name in kong didn't match the name in the config.